app/controllers/events-controller.js

```diff
@@ -0,0 +1,78 @@
+const { Event } = require("../models");
+const utils = require("../utils");
+
+const get_all_events = async (req, res) => {
+  await Event.findAll().then((result) => {

```

Comment:
buna pagination yapmak lazim. bu sekilde uygulamayi yamulturuz. simdiden onlem almak gerekli.

Reply:
+1, Simdiden pagination in param handlerlari gozumde canlanir gibi oldu :D